NuRadioReco/examples/example_data/dual_polerized_antenna_station.json

```diff
@@ -0,0 +1,44 @@
+{
+    "_default": {},
+    "channels": {
+        "0": {
+            "ant_comment": "dipolarized antenna channel east",
+            "ant_orientation_phi": 0.0,
+            "ant_orientation_theta": 0.0,
+            "ant_type": "createLPDA_100MHz_InfAir",
+            "ant_position_x": 0.0,
+            "ant_position_y": -0.0,
+            "ant_position_z": 1,
```

Comment:
Is there a specific reason to make the z-positions 1 instead of 0, just like x and y?

Reply:
Well antennas are typically not lying on the ground :d

Reply:
Oh, you the z-coordinate to define the top of the antenna? I usually thought of it as the base of the antenna. Though I just checked the documentation and it does say the coordinates define the location of the "feed point" of the antenna, which in case of the SKALA's I guess are at the top

Reply:
Honestly I do not care. It has no effect